### ISSUE TITLE
Incompatible with Python 3

### ISSUE THREAD

**Issue by hamotoJohn** [mdsol/rwslib#22]
This is not a problem for me, since I can easily use Python 2, but it's worth noting. Python 3 does not work with this library as packaged on PyPi. The main problem appears to be ImportErrors for `urlencode`. Python 3 has a different import path. I think the following resolves it:
```python
try:
    from urllib import urlencode
except ImportError:
    from urllib.parse import urlencode
```
There are also some Python2 `print` statements--as opposed to the Python3 `print()` function. Converting those to the function syntax should work for Python2 and Python3.

There may be other problems, but I have not had time to delve into it. My guess is that there could be some problems around encoding, since Python3 handles unicode text vs bytes somewhat differently. Perhaps all of this could be shimmed into place with the `six` module.

**Comment by isparks** [comment]
We're python 2 users so this hasn't been an issue for us. Would you prefer to use python3? We want to be good citizens so we could certainly take a look at it.

**Comment by isparks** [comment]
@hamotoJohn this was now fixed in 1.0.8, available on PyPi. https://pypi.python.org/pypi/rwslib 

Special thanks to my colleague @glow-mdsol who did most of the legwork. Py 3.4 and PyPy support were added.
